modules/component/src/componentNext.ts

```diff
@@ -69,7 +33,17 @@ type Comparator = <S, P>(s1: S, p1: P, s2?: S, p2?: P) => boolean
 const defaultComparator: Comparator = () => true
 
 const arg2 = <A, B>(a: A, b: B) => b
+
+/**
+ * Type safe, composable, and view agnostic component API
+ */
 export class ComponentNext<P1 extends ComponentProps> {
+  /**
+   * @param _init Function which return initial state of the component
```

Comment:
use `returns`... so that there's consistency across the other params

Reply:
Comment for return value of constructor is redundant IMO